include/swift/Runtime/Config.h

```diff
@@ -122,7 +122,7 @@
 #define SWIFT_LLVM_CC(CC) SWIFT_LLVM_CC_##CC
 
 // Currently, RuntimeFunction.def uses the following calling conventions:
```

Comment:
While you're here, can you correct this misspelling? The file name is `RuntimeFunctions.def`.